src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,719 @@
+      $ref: ./fields.yaml#/in_operator
+    employee_user_ids:
+      type: array
+      description: |
+        Employee user ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - usabcds1
+        - usabcds2
+    employee_user_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_titles:
+      type: array
+      description: |
+        Employee titles targeted by this policy.
+      items:
+        type: string
+      example:
+        - CEO
+        - Manager
+    employee_titles_op:
+      $ref: ./fields.yaml#/in_operator
+    action_flag:
+      type: boolean
+      description: |
+        Whether to flag the expense when it violates policy.
+      example: True
+    action_show_warning:
+      type: boolean
+      description: |
+        Whether to show warning to employee when the expense that they are creating violates the policy.
+      example: True
+    action_make_unreportable:
+      type: boolean
+      description: |
+        Whether to make the expense unreportable when it violates policy. <br>
+        (Expense will be created but employee will not be able to add it to report).
+      example: True
+    action_remove_employee_approver1:
+      type: boolean
+      description: |
+        Whether to remove the primary approver from the report when the expense violates policy.
+      example: True
+    action_add_employee_approver2:
+      type: boolean
+      description: |
+        Whether to add secondary approver 1 as approver to report when the expense violates policy.
+      example: True
+    action_add_employee_approver3:
+      type: boolean
+      description: |
+        Whether to add secondary approver 2 as approver to report when the expense violates policy.
+      example: True
+    action_add_project_approver1:
+      type: boolean
+      description: |
+        Whether to add project approver 1 as approver to report when the expense violates policy.
+      example: True
+    action_add_project_approver2:
+      type: boolean
+      description: |
+        Whether to add project approver 2 as approver to report when the expense violates policy.
+      example: True
+    action_add_department_head_approver:
+      type: boolean
+      description: |
+        Whether to add department head as approver to report when the expense violates policy.
+      example: True
+    action_cap_amount:
+      type: boolean
+      description: |
+        Whether to cap the amount to expense_amount_max when the expense violates policy.
+      example: True
+    action_add_approver_user_ids:
+      type: array
+      description: |
+        List of approver user ids to be added to report when the expense violates policy.
+      items:
+        type: string
+      example:
+        - usabcds3
+        - usabcds4
+    action_override_distance_slab_rate:
+      type: number
+      description: |
+        Override the distance slab rate with the set value when the expense violates policy.
```

Comment:
explain distance slab rate by either adding hyperlink to someplace or adding text